README.md

```diff
@@ -1,221 +1,172 @@
 
-If you’d like to develop the Frank!Flow it would be handy to store the Frank!Flow project in a different location than your franks. To tell the Frank!Runner that our Frank!Flow project is located at a different location, we need to add an extra argument to the build.xml. This extra argument can be seen on line 6:
+##### Backend
 
-```XML
-<project default="restart-frank2frank-flow">
-	<target name="restart-frank2frank-flow">
-		<basename property="project.dir" file="${basedir}"/>
-		<exec executable="../frank-runner/restart.bat" vmlauncher="false" failonerror="true">
-			<arg value="-Dproject.dir=${project.dir}"/>
-			<arg value="-Dfrank-flow.frontend.path=C:\PATH_TO_YOUR_FRANK-FLOW_PROJECT\frank-flow\src\main\resources\frontend"/>
-		</exec>
-	</target>
-</project>
-```
+- [Java](https://www.java.com/nl/download/)
+- [Maven](http://maven.apache.org/)
 
-You should edit `PATH_TO_YOUR_FRANK-FLOW_PROJECT` to the correct location. As you can see, the rest of the path contains the location to the frontend distribution folder.
+##### Frontend
 
-#### Restart
+- [Node.js](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/getting-started/install)
 
-The last file is optional, because it is up to you how you restart your Frank!Runner. Some people like to use the build in Task Manager of there IDE, which will have a restart task because of the build.xml.
+#### DevTools
 
-You could also create an additional `restart.bat` so it can be called via the command line interface (or double click). This file should contain:
+There are some tools that have to be used during development. These tools are needed for linters that keep the code clean and formatted the same way. The tools get run automatically by a Git hook.
 
-```bash
-call ..\frank-runner\ant.bat
-if %errorlevel% equ 0 goto end
-rem https://superuser.com/questions/527898/how-to-pause-only-if-executing-in-a-new-window
-set arg0=%0
-if [%arg0:~2,1%]==[:] if not [%TERM_PROGRAM%] == [vscode] pause
-:end
-```
+To install the tools go to the folder `frank-flow/src/frontend` this is needed because of the way this mono repo has been structured. This might change in the feature. Run `yarn` in this folder and everything should be good to go!
 
-### Frank!Flow
+### Frontend
 
-Clone this repository (or a fork) to start developing the Frank!Flow. This project consists of two main parts: The JavaScript frontend and the Java backend.
+The code for the frontend is located at `frank-flow/src/frontend`. This folder contains the cypress integration tests, the source files for the application and configurations. 
 
-#### Requirements
+#### Building the frontend
 
-There are some basic requirements that are needed to test or develop the Frank!Flow application. These requirements are:
+There are two ways to build the application, for development and for production. The difference lays in the way the code gets transpiled.
 
```

Comment:
What does "transpile" mean? Can you explain this a bit?

Reply:
I think that is out of the scope of this document. I will change it to something more understandable.

Reply:
Ok.